### PR TITLE
revert all recent printing commits

### PR DESCRIPTION
**Pull request by cg505**
Revert "Partial revert of 27a120"

This reverts commit da8b9279e070bec4111908809ed34a53657a24d1.

Revert "temporarily remove local PPDs because of multiple copy issues"

This reverts commit 4e92142d07ac316c18a86d889649326e4d9734bc.

Revert "Add NOPASSWD when giving root to individual users"

This reverts commit 7083e9e94e65fe41990fdef879d4ead0d724cc62.

Revert "Add size checks to raster-filter"

This reverts commit 66ae9e49ffbf387b34f6c1c1142ba1b30ea17e16.

Revert "clear cups queues and re-enable printers on login"

This reverts commit b32b453e484d90c9a0c9a2e6b8ad8fb492db804c.

Revert "add ppd files for local printer with page size and duplex info"

This reverts commit 684ac76cee2a62a5cf4200228cd85ec86831fcc6.

Revert "Revert "Removing printhost check for Google Docs" (#740)"

This reverts commit 34f75e80e1b1475e099e5f1fa189a753001113fc.

Revert "use fqdn for ipp (#734)"

This reverts commit 0c594c478c0fdd234e4917990bf93bd8f978c43a.

Revert "Tweaked raster-filter if statements"

This reverts commit 774155238540d7cf5682d794635faf0f16f4e21f.

Revert "Some tweaks to convert_failure"

This reverts commit 56bd453ea7d7743cc39630df0abdb59368dd574b.

Revert "Improved raster-filter, convert_failure"

This reverts commit 4ae1e2eeaeb6c07f8a78e38c89e175bfea1e1d8f.

Revert "Make raster-filter email convert error message to root mailing list"

This reverts commit d103a384630a5d766afd48b3b4a15478e4419a6d.

Revert "Remove libimage-exiftool-perl from packages.pp"

This reverts commit 1e8f81a9c9b9bd4854e15e34530b8c9c10adbe05.

Revert "Small change to raster-filter"

This reverts commit 0031c19f4c9d3417975f499f11750788cb89a699.

Revert "Removing printhost check for Google Docs"

This reverts commit c055f88e1fc7a851f97c6cd4c6fe9cb74fdc22fa.

Revert "Further removing old PDF rasterization setup"

This reverts commit 1b5fcf6220a9e6aab6f4206d6fa90adb6bc954a9.

Revert "Changing browser settings to open PDFs in-browser"

This reverts commit 27a120c3e358a8c4f5cae710f0444cc492fdca63.

Revert "Remove PDF rasterization upon clicking on a PDF"

This reverts commit 8c362102271989dda49723b342faf979ee65e6e1.

Revert "Changing ImageMagick settings to give more memory"

This reverts commit 39d6970ee45c4c817f7af396242d989893d51ba3.

Revert "New CUPS settings, to have PDFs rasterized before printing"

This reverts commit cabb0ac357d4a1b8a94ff18488b77e074e6ffdcf.

### CODE REVIEW

**Comment by fydai** [comment]
I approve presuming this is tested.

File: modules/ocf/manifests/packages/cups.pp [hunk 1]
```diff
@@ -1,50 +1,20 @@
...
 
   # install cups
-  package { [ 'cups', 'cups-bsd', 'cups-tea4cups' ]: }
+  package { [ 'cups', 'cups-bsd' ]: }
 
   file {
     # set print server destination
     '/etc/cups/client.conf':
-      content => "ServerName localhost\nEncryption Always\n",
+      content => "ServerName printhost\nEncryption Always\n",
```
Comment (abizer) [inline modules/ocf/manifests/packages/cups.pp]: does this need to be the fqdn

File: modules/ocf/templates/sudoers.erb [hunk 2]
```diff
@@ -16,10 +16,10 @@ ocfdeploy ALL=NOPASSWD: /usr/local/sbin/puppet-trigger
 # User specification
 %ocfroot ALL=(ALL) <%= @nopasswd ? 'NOPASSWD:' : '' %> ALL
 <% @gsudo.each do |group| -%>
-%<%= group %> ALL=(ALL) NOPASSWD: ALL
+%<%= group %> ALL=(ALL) <%= @nopasswd ? 'NOPASSWD:' : '' %> ALL
```
Comment (abizer) [inline modules/ocf/templates/sudoers.erb]: this seems like an errant revert

File: modules/ocf/manifests/packages/cups.pp (hunk 1 above)
Comment (cg505) [reply modules/ocf/manifests/packages/cups.pp]: no

File: modules/ocf/templates/sudoers.erb (hunk 2 above)
Comment (cg505) [reply modules/ocf/templates/sudoers.erb]: ty

**Pull request merged**